mobilebot/mobilebot.c

```diff
@@ -211,10 +223,11 @@ void publish_mb_msgs(){
 void mobilebot_controller(){
     update_now();
     read_mb_sensors();
+    mb_update_odometry(&mb_odometry, &mb_state);
     publish_mb_msgs();
-
+    
     mb_controller_update(&mb_state, &mb_setpoints);
-    mb_motor_set(RIGHT_MOTOR, mb_state.right_cmd);
+    mb_motor_set(RIGHT_MOTOR, -mb_state.right_cmd);
```

Comment:
fix this with motor polarity definitions later?

Reply:
Yes